app/src/main/java/net/skyscanner/backpack/demo/compose/BpkStarRatingStory.kt

```diff
@@ -0,0 +1,175 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import net.skyscanner.backpack.compose.starrating.BpkHotelRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRatingSize
+import net.skyscanner.backpack.compose.starrating.RoundingType
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun BpkStarRatingStory(
+  modifier: Modifier = Modifier,
+) {
+  val context = LocalContext.current
+  Box(
+    modifier = modifier.fillMaxHeight(),
+    contentAlignment = Alignment.Center,
+  ) {
+    Column(modifier = Modifier.padding(start = BpkSpacing.Lg)) {
+      BpkText(
+        text = stringResource(R.string.static_default_rating_title),
+        style = BpkTheme.typography.heading2,
+      )
+      Row() {
+        Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Large,
+            text = stringResource(R.string.icons_large),
+          )
+        }
+        Column() {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Small,
+            text = stringResource(R.string.icons_small),
+          )
+        }
+      }
+      Spacer(modifier = Modifier.padding(top = BpkSpacing.Lg))
+      Column() {
+        BpkText(
+          text = stringResource(R.string.hotel_star_rating_title),
+          style = BpkTheme.typography.heading2,
+        )
+        Row() {
+          Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Large,
+              text = stringResource(R.string.icons_large),
+            )
+          }
+          Column() {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Small,
+              text = stringResource(R.string.icons_small),
+            )
+          }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun HotelRatingSample(context: Context, size: BpkStarRatingSize, text: String) {
+  BpkText(
+    text = text,
+    style = BpkTheme.typography.heading3,
+  )
+  BpkHotelRating(
+    rating = 1f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 1f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 2f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 2f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 3f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 3f, 5),
```

Comment:
What happened with the lambda for contentDescription? It seems we still duplicating the way we calculate the logic here

Reply:
I think @marianeum suggested a lambda was not necessary here and we came to this solution. By duplicating do you mean setting the rating in the rating attribute but also setting it inside the contentDescription?

Reply:
Yes exactly. We can use string placeholders instead of lambda here, as we do with View version